src/methods/user/stats.js

```diff
@@ -0,0 +1,15 @@
+export default function fetch(params) {
```

Comment:
I agree this needs to be renamed, by calling it `fetch` it could introduce mis-understanding around the interface as developers may assume it is the global fetch API we are calling.